app/views/energy_tariffs/energy_tariffs/show.html.erb

```diff
@@ -1,20 +1,79 @@
-<h1><%= t('schools.user_tariffs.show.title') %></h1>
+<h1><%= @energy_tariff.name %></h1>
 
 <div class="energy_tariff">
 
   <% if @energy_tariff.tariff_holder.school? %>
     <p>
-      <%= t('schools.user_tariffs.show.introduction_html', electricity_analysis_path: analysis_page_finder_path(urn: @tariff_holder.urn, analysis_class: 'AdviceElectricityCosts'), gas_analysis_path: analysis_page_finder_path(urn: @tariff_holder.urn, analysis_class: 'AdviceGasCosts')) %>.
+      <%= t('schools.user_tariffs.show.introduction_html', analysis_path: analysis_page_finder_path(urn: @tariff_holder.urn, analysis_class: cost_analysis_class_for(@energy_tariff.meter_type)), meter_type: t("common.#{@energy_tariff.meter_type}").downcase) %>.
     </p>
     <br/>
   <% end %>
 
-  <%= render 'energy_tariffs/energy_tariffs/header', energy_tariff: @energy_tariff %>
+  <div class="d-flex justify-content-between">
+    <div>
+      <h4><%= t('schools.user_tariffs.show.dates.title') %></h4>
+    </div>
+    <div>
+      <%= link_to t('common.labels.edit'),
```

Comment:
Where we've allowed editing of dates and meters can you add a system spec. Could write a test that starts from having an existing saved tariff and then confirm you can a) edit meters (for schools only) and b) edit title/dates (should apply to all tariff holders)

Reply:
:+1: editing functionality tests added for a & b to the relevant shared examples.